qemu/tests/cfg/block_commit.cfg

```diff
@@ -18,6 +18,18 @@
     base_image = "images/sn1"
     top_image = "images/sn3"
     expected_image_file = "images/sn1"
+    variants:
+        - @default:
+        - with_data_plane:
+            only Host_RHEL.m7
+            only virtio_blk virtio_scsi
+            #Add a redundant iothread for testing purpose
+            iothreads = "iothread0 iothread1"
```

Comment:
Hi qizhu, what do you mean by "Add a redundant iothread for testing purpose"?  "iothreads = "iothread0 iothread1" is not used?

Reply:
@YanhuiMa The redundant one indicates "iothread1", which is not used. Only iothread0 used by following lines.  This is designed after query data plane feature owner, @aliang123 , who told me that a redundant iothread might trigger some unexpected bz.

Reply:
Checked all cfgs and test run correctly with those configured params.
Acked-by: Aihua Liang <aliang@redhat.com>